web/src/app/modules/shared/components/presentation/datagrid/datagrid.component.ts

```diff
@@ -54,6 +56,7 @@ export class DatagridComponent
   buttonGroup?: ButtonGroupView;
   isModalOpen = false;
   defaultPageSize: number;
+  detail = { name: 'test' };
```

Comment:
`test` is not a good default value

Reply:
This line was a leftover from development and is not needed.